Doc/library/sndhdr.rst

```diff
@@ -54,3 +54,52 @@ be the sample size in bits or ``'A'`` for A-LAW or ``'U'`` for u-LAW.
    .. versionchanged:: 3.5
       Result changed from a tuple to a namedtuple.
 
+The following sound header types are recognized, as listed below with the return value
+from :func:`whathdr`: and :func:`what`:
+
++------------+------------------------------------+
+| Value      | Sound header format                |
++============+====================================+
+| ``'aifc'`` | Compressed Audio Interchange Files |
++------------+------------------------------------+
+| ``'aiff'`` | Audio Interchange Files            |
++------------+------------------------------------+
+| ``'au'``   | Au Files                           |
++------------+------------------------------------+
+| ``'hcom'`` | HCOM Files                         |
++------------+------------------------------------+
++------------+------------------------------------+
```

Comment:
Why the empty row in the table?

Reply:
I just checked main, its not there, maybe another fix I dont know fixed it.

Reply:
How so?  I see it here https://github.com/python/cpython/blob/main/Doc/library/sndhdr.rst (and in raw view)

Reply:
I think I checked out the wring code version locally, my bad. I have a new PR to fix that : https://github.com/python/cpython/pull/98337